### ISSUE TITLE
Are only card payment allowed?

### ISSUE THREAD

**Issue by popekabu** [tolu360/react-native-paystack#61]
I would like to ask if mobile money can be done with this plugin because in the examples all i see are cards. Also finally is conversion handled by this plugin or how best can i do that

**Comment by tolu-paystack** [comment]
Yeah, the SDK just charges cards. For other payment methods, you'll need to integrate to Paystack's API directly
